src/calls/calls.controller.ts

```diff
@@ -22,4 +23,14 @@ export class CallsController {
   ): Promise<Call> {
     return await this.callsService.createCall(restaurantId, createCallRequestDto);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @Put(':restaurant_id/:call_id')
+  async updateCategory(
```

Comment:
코드를 가져올때는 요런 부분을 신경써주세요!
```suggestion
  async updateCall(
```